cluster-autoscaler/simulator/drainability/context.go

```diff
@@ -17,12 +17,17 @@ limitations under the License.
 package drainability
 
 import (
+	"time"
+
 	"k8s.io/autoscaler/cluster-autoscaler/core/scaledown/pdb"
 	"k8s.io/autoscaler/cluster-autoscaler/simulator/options"
+	kube_util "k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes"
 )
 
 // DrainContext contains parameters for drainability rules.
 type DrainContext struct {
 	RemainingPdbTracker pdb.RemainingPdbTracker
 	DeleteOptions       options.NodeDeleteOptions
```

Comment:
I'd make `DeleteOptions` a param to `rules.Default` rather than part of the context. That way we can construct a set of rules once and keep it unchanged later. All the values in that struct are set once at initialization and then never changed afterwards.

Reply:
This is a good idea. Done.